README.md

```diff
@@ -0,0 +1,109 @@
+About the Allergy API Module
+===
+The Allergy API is a Java-based API that handles patient data in OpenMRS related to allergies. More specifically, the main motivations
+behind this API are to easily add allergy objects to patients, to change certain aspects of allergy objects for a patient such as severity and allergens in an organized fashion
+(through variables such as allergy ID), and to compare allergies with each other to see if they have similar properties or characteristics.
+
+Developer Documentation
+---
+
+Allergy.java

```

Comment:
We don't need this much of the developer information for each JAVA file. As we need to change this ReadMe when we change those functions. We have method level comments to show the necessary docs. 

Can you simply mention some common facts for Developer documentation? Like JIRA links and other folder structure.